proxywasm/hostcall.go

```diff
@@ -82,6 +82,22 @@ func GetHttpCallResponseTrailers() (types.Trailers, error) {
 	return ret, types.StatusToError(st)
 }
 
+func CallForeignFunction(funcName string, param string) (ret string, err error) {
```

Comment:
let's have []byte as a type for param which is more generic 
```suggestion
func CallForeignFunction(funcName string, param []byte) (ret string, err error) {
```

Reply:
and also have the return to be []byte as well
```suggestion
func CallForeignFunction(funcName string, param string) (ret []byte, err error) {
```